packages/workflow/src/logs.ts

```diff
@@ -77,26 +77,34 @@ export const log: WorkflowLogger = Object.fromEntries(
 
 export function executeWithLifecycleLogging(fn: () => Promise<unknown>): Promise<unknown> {
   log.debug('Workflow started');
+  const activator = assertInWorkflowContext('Workflow.executeWithLifecycleLogging(...) may only be used from workflow context.');
```

Comment:
Nit: Just use `getActivator()` here instead of `assertInWorkflowContext`. `executeWithLifecycleLogging` is not exposed publicly, so essentially no way that a user would call it improperly.

Reply:
@mjameswh done